### PR TITLE
Reader: ensure Gutenberg tables are visually similar to the original site

### PR DESCRIPTION
**Pull request by tophertoy**
Part of #43595.

#### Changes proposed in this Pull Request

This resolves the visual differences with the tables in reader as requested in #43595

#### Testing instructions

http://calypso.localhost:3000/read/blogs/165243437/posts/1717

**Before**
Screenshot 2020-07-20 17 02 48

**After**

Screenshot 2020-07-22 14 49 29

### CODE REVIEW

**Comment by tophertoy** [comment]
@bluefuton any thoughts on anything extra you'd like to see in reader for this one?

**Comment by bluefuton** [comment]
@tophertoy can we visually distinguish `th` somehow? Either by bolding the text, or adding a background, or both?

**Comment by tophertoy** [comment]
> @tophertoy can we visually distinguish `th` somehow? Either by bolding the text, or adding a background, or both?

Sure thing. New after image is above. Could be too subtle..?

File: client/blocks/reader-full-post/_content.scss [hunk 1]
```diff
@@ -265,6 +265,22 @@
 		border-color: var( --color-neutral-10 );
 	}
 
+	.wp-block-table table {
+		border-collapse: collapse;
+	}
+
+	.wp-block-table td,
+	.wp-block-table th {
+		border: 1px solid #767676;
```
Comment (bluefuton) [inline client/blocks/reader-full-post/_content.scss:274]: Is there a suitable `var` for this colour?

Comment (tophertoy) [reply client/blocks/reader-full-post/_content.scss:274]: Do you use this source code to find these? https://github.com/Automattic/color-studio

Comment (tophertoy) [reply client/blocks/reader-full-post/_content.scss]: done.

**Pull request merged**